pimd/pim_pim.c

```diff
@@ -253,6 +253,19 @@ int pim_pim_packet(struct interface *ifp, uint8_t *buf, size_t len)
 					      pim_msg_len - PIM_MSG_HEADER_LEN);
 		break;
 	case PIM_MSG_TYPE_JOIN_PRUNE:
+		if (ip_hdr->ip_dst.s_addr != qpim_all_pim_routers_addr.s_addr) {
```

Comment:
make this a function and add this too assert messages being received too?

Reply:
hello should be this as well?

Reply:
Yes that was coming up anyways in the next batch. Will add it to this PR it self.